python/test/t_LinearModelAlgorithm_std.py

```diff
@@ -10,40 +10,44 @@
 try:
 
     # lm build
-    print("Fit y ~ 3 - 2 x + 0.05 * sin(x) model using 20 points (sin(x) ~ noise)")
+    print(
+        "Fit y ~ 3 - 2 x + 0.05 * sin(x) model using 20 points (sin(x) ~ noise)")
     size = 20
     oneSample = ot.Sample(size, 1)
     twoSample = ot.Sample(size, 1)
     for i in range(size):
-        oneSample[i,0] = 7.0 * sin(-3.5 + (6.5 * i) / (size - 1.0)) + 2.0
-        twoSample[i,0] = -2.0 * oneSample[i, 0] + 3.0 + 0.05 * sin(oneSample[i, 0])
+        oneSample[i, 0] = 7.0 * sin(-3.5 + (6.5 * i) / (size - 1.0)) + 2.0
+        twoSample[i, 0] = -2.0 * oneSample[
+            i, 0] + 3.0 + 0.05 * sin(oneSample[i, 0])
```

Comment:
Maybe your indent script should skip python/test directory?  These changes are questionable.